docs/user-management.rst

```diff
@@ -164,16 +164,17 @@ Updating a user works exactly as creating a new user, except that the ``uid`` pr
 
 .. code-block:: php
 
+    $uid = 'some-uid';
     $properties = [
         'displayName' => 'New display name'
     ];
 
-    $updatedUser = $auth->updateUser($properties);
+    $updatedUser = $auth->updateUser($uid,$properties);
 
     $request = \Kreait\Auth\Request\UpdateUser::new()
         ->withDisplayName('New display name');
 
-    $updatedUser = $auth->updateUser($request);
+    $updatedUser = $auth->updateUser($uid,$request);
```

Comment:
Please add a space after the comma (the small things 😅)

Reply:
:-). Doing now.